clouddq/templates/dbt/macros/macros.sql

```diff
@@ -20,9 +20,6 @@
     '{{ fully_qualified_table_name }}' AS table_id,
     '{{ column_name }}' AS column_id,
     data.{{ column_name }} AS column_value,
-    {% for ref_column_name in include_reference_columns %}
-        data.{{ ref_column_name }} AS {{ ref_column_name }},
-    {%- endfor -%}
```

Comment:
Are changes to simple rule validation intended?

Reply:
Yes, because when both simple rule and complex rule run in the same rule_binding. The rule_binding sql will have different number of columns in the union that will cause the execution to fail.